multiaccounts/settings/columns.go

```diff
@@ -436,6 +436,11 @@ var (
 		dBColumnName:   "address",
 		valueHandler:   AddressHandler,
 	}
+	IncludeWatchOnlyAccount = SettingField{
```

Comment:
@Khushboo-dev-cpp this is something we need to sync among devices and backup to waku as well, mechanisms which do that are already in place, so please just set `syncProtobufFactory` appropriately.